templates/partials/answers/address.html

```diff
@@ -44,6 +44,8 @@
     "ariaOneResult": _("There is one suggestion available."),
     "ariaNResults": _("There are {n} suggestions available."),
     "ariaLimitedResults": _("Results have been limited to 10 suggestions. Type more characters to improve your search"),
+    "ariaGroupedResults": _("There are {n} for {x}"),
```

Comment:
I think this needs to be `There are {n} addresses for {x}`, as I think this would resolve to something like `There are 5 for CF23`.

https://github.com/ONSdigital/design-system/blob/61db1aaadca98deca77e7ee40d1dd15654024278/src/components/input/autosuggest/autosuggest.ui.js#L433

Reply:
So @MebinAbraham  has investigated and this is correct - it uses the `groupCount` property interpolated into the `ariaGroupedResults` message. We shouldn't really be doing that, but that would need to be fixed in the design system.